yarn.lock

```diff
@@ -2580,6 +2577,32 @@
     "@types/jquery" "*"
     "@types/rsvp" "*"
 
+"@types/ember@^3.16.5":
```

Comment:
It may be worth running a `yarn deduplicate` here or otherwise forcing a resolution; `@types/ember-testing-helpers` appears (per previous entry in the lock file) to be resolving `@types/ember@3.16.4`. Given CI is happy and your local build was as well, it seems like that's not currently causing an issue, but I have been bitten enough times that I always make sure there's only a single resolved version of any `@types/*` package where possible.

Reply:
done